Updated german translations

map-per opened the pull request:
Updated the german translations in types_strings.txt

biodranik commented on data/strings/types_strings.txt:
```diff
@@ -3852,8 +3852,9 @@
 
   [type.amenity.taxi]
-    en = Taxi
+    en = Taxi Rank
```
```suggestion
    en = Taxi Stand
    en-GB = Taxi Rank
```

Mr-Mime reviewed (approved):
LGTM.

Thank you @map-per for you contribution :muscle:

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -14712,7 +14715,7 @@
     cs = Komunikační věž
     da = Kommunikationstårn
-    de = Kommunikationsturm
+    de = Mobilfunkmast
```
[tower:type=communication](https://wiki.openstreetmap.org/wiki/DE:Tag:tower:type%3Dcommunication) doesn't necessarily mean its a cell tower, it could as well be a radio or television tower. Therefore I would keep "Kommunikationsturm".

j13m126 replied:
or change it to Funkmast/Funkturm as it is called in the Wiki

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -11849,7 +11851,7 @@
     cs = Palác
     da = Palads
-    de = Palais
+    de = Palast
```
I personally prefer 'Palais' and looking at overpass I see that almost all castles in Germany and Austria are called Palais x rather than Palast. What do you think @Mr-Mime ?

map-per replied:
Looks like in Germany most castle:type=palaces are called 'Schloss'. In Austria however 'Palais' is common.

biodranik replied:
@map-per yes, you can start adding de-AT, we will later support it.

j13m126 replied:
Sure, but there's no official german translation for Buckingham Palace. ;)
I mean, just look at the wikipedia list for [Germany](https://de.wikipedia.org/wiki/Kategorie:Palast_in_Deutschland) and [Austria](https://de.wikipedia.org/wiki/Kategorie:Palais_in_%C3%96sterreich). There's not a single palace called 'Palast'. That's why it would sound unnatural to me to call any of them Palast.

j13m126 replied:
But we can let the majority decide, we don't need to differentiate between de and de-AT for that.

map-per replied:
> How would you call the Buckingham Palace in German? 'Palast' or 'Palais'?

What I wanted to say is: The translation will be used in the entire world. Would you also call a palace outsiede of Germany, Austria and Switzerland 'Palais'?

telluroid commented on data/strings/types_strings.txt:
```diff
@@ -26936,7 +26939,7 @@
   [type.piste_type.downhill]
     en = Downhill Ski Run
-    de = Abfahrt
+    de = Ski Abfahrt
```
Ski-Abfahrt

Mr-Mime replied:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Ski-Abfahrt
```

telluroid commented on data/strings/types_strings.txt:
```diff
@@ -26951,7 +26954,7 @@
     ref = type.piste_type.downhill
     en = Advanced Downhill Ski Run
-    de = Schwierige Abfahrt
+    de = Schwierige Ski Abfahrt
```
Ski-Abfahrt

biodranik replied:
No, it is important to distinguish the complexity. Users can check it by a long-tap.

telluroid replied:
You got me wrong. I wanted to say that the hyphen is missing. Sorry for not beeing clear. In this exact case it should say:
Schwierige Ski-Abfahrt
Without hyphen it is grammatically wrong.

biodranik replied:
Thank you for the clarification!

Mr-Mime replied:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Schwierige Ski-Abfahrt
```

telluroid commented on data/strings/types_strings.txt:
```diff
@@ -26963,7 +26966,7 @@
     ref = type.piste_type.downhill
     en = Easy Downhill Ski Run
-    de = Leichte Abfahrt
+    de = Leichte Ski Abfahrt
```
Ski-Abfahrt

telluroid commented on data/strings/types_strings.txt:
```diff
@@ -26975,7 +26978,7 @@
     ref = type.piste_type.downhill
     en = Expert Downhill Ski Run
-    de = Profi Abfahrt
+    de = Profi Ski Abfahrt
```
Ski-Abfahrt

Mr-Mime replied:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Profi-Ski-Abfahrt
```

telluroid commented on data/strings/types_strings.txt:
```diff
@@ -26996,7 +26999,7 @@
     ref = type.piste_type.downhill
     en = Intermediate Downhill Ski Run
-    de = Mittelschwere Abfahrt
+    de = Mittelschwere Ski Abfahrt
```
Ski-Abfahrt

Mr-Mime replied:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Mittelschwere Ski-Abfahrt
```

telluroid commented on data/strings/types_strings.txt:
```diff
@@ -27008,7 +27011,7 @@
     ref = type.piste_type.downhill
     en = Novice Downhill Ski Run
-    de = Anfängerfreundliche Abfahrt
+    de = Anfängerfreundliche Ski Abfahrt
```
Ski-Abfahrt

map-per commented:
Thanks for all the feedback!
I updated the PR to include the suggestions. I sticked to 'Palast' instead of 'Palais' however ('Palast' is way more common). Moreover I added 'Fähranleger' as the translation for a ferry terminal

biodranik commented on data/strings/types_strings.txt:
```diff
@@ -3852,8 +3852,10 @@
     zh-Hant = 淋浴間
 
   [type.amenity.taxi]
```
If we already touched it, can you please later update all translations for the taxi (and in categories.txt) in a separate PR?
```
en:Taxi Stand
ar:محطة التاكسي
be:Стаянка таксі
bg:Стойка за таксита
ca:Parada de taxis
cs:Stanoviště taxi
da:Taxaholdeplads
de:Taxistand
el:Περίπτερο ταξί
es:Parada de taxis
es-MX:Parada de taxi
et:Taksopeatus
eu:Taxi geltokia
fa:ایستگاه تاکسی
fi:Taksitolppa
fr:Station de taxis
he:תחנת מוניות
hu:Taxi stand
id:Tempat taksi
it:Stazione dei taxi
ja:タクシースタンド
ko:택시 승차장
lt:Taksi stendas
mr:टॅक्सी स्टँड
nb:Drosjeholdeplass
nl:Taxistandplaats
pl:Postój taksówek
pt:Ponto de táxi
ro:Stație de taxi
ru:Стоянка такси
sk:Stanovište taxíkov
sv:Taxistation
sw:Stendi ya teksi
th:ป้ายรอแท็กซี่
tr:Taksi durağı
uk:Стоянка таксі
vi:Đón taxi
zh-Hans:出租车站
zh-Hant:的士站
```

biodranik replied:
@vng will taxi be searchable by a single `taxi` word in all these cases above?

map-per replied:
Ok, will do that. 
From a quick test it looks like `taxi` should probably be searchable in other languages as well if one uses: `en:3Taxi|Taxi Stand`

map-per replied:
I struggled to build the project with QT6 and gave up for now, so I won't be able to change the taxi translations and test the search in time for the next release.

biodranik replied:
@map-per what kind of build errors did you get?

map-per replied:
I'm getting this error:
```
map-per@Linux-PC:~/Dokumente/organicmaps$ tools/unix/build_omim.sh -r desktop
-- Using compiler GNU 11.3.0
-- Using Unity Build with batch 50, export UNITY_DISABLE=1 or use -DUNITY_DISABLE=ON to disable it.
-- export COLORS_DISABLE=1 or use -DCOLORS_DISABLE=ON to disable colored compiler output.
Setting PLATFORM_LINUX to true
-- Build type: Release
-- Using ld.gold
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt6/QtFeature.cmake:1212 (message):
  Feature rpath is already defined to be "ON" and should now be set to "OFF"
  when importing features from Qt6::Core.
Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/Qt6Core/Qt6CoreConfig.cmake:128 (qt_make_features_available)
  /usr/lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake:219 (find_package)
  CMakeLists.txt:244 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/map-per/Dokumente/omim-build-release/CMakeFiles/CMakeOutput.log".
```

Maybe I messed up something while trying to install Qt6. Should the Installation work with a clean Ubuntu22 Installation? If so I should maybe reset the computer.

biodranik reviewed (approved):
Thanks! AFAIR, changes here should also be reflected in categories.txt for these terms to be searched. You can leave an old and new translation in categories.txt so users can find both, and add a number to display hints in the search suggestions.

map-per commented:
Ok, added the new strings to categories.txt

biodranik commented on data/categories.txt:
```diff
@@ -3005,7 +3005,7 @@ da:Tempel|kultsted
 fi:Temppeli|uskonnollinen kohde
 fr:Temple|lieu de culte
-de:Tempel|Anbetungsstätte
+de:Hinduistischer Tempel|Tempel|Anbetungsstätte
```
Can we create a @temple (Tempel) alias and use it in concrete subtypes of temples, to avoid copy-pasting the same "Tempel" everywhere (in a separate PR, of course)?

map-per replied:
Counted 8 different places of worship, so that should be worth it. I'd rather name it @**place_of_worship** however.

biodranik reviewed (approved):
Thank you very much!
